application/views/template.php

```diff
@@ -45,12 +45,21 @@
 								echo $content;
 							}
 						} else {
-							echo 'Page does not have any content';
-						}
+							$services_down = 'services LMD and Naemon are not running';
 
+							// Check LMD.
+							exec('service lmd status', $output, $return_var_lmd);
```

Comment:
This will always return on non zero on 7.5. Doesn't make sense to do this check in 7.5 as we do not have LMD.

Reply:
So should we remove this branch?

Reply:
On Monitor 7.5 I guess it is safe to say that when this page shows, it means that the Naemon service is not alive. So we can probably just write that directly without further checking.

Reply:
The "problem" here is that page is only shown when LMD is down but if both LMD and Naemon are down, the banner will say both are down. (At least in Monitor 8)

Reply:
LMD isn't included in Monitor 7 at all.